extension/js/common/core/crypto/pgp/openpgp-key.ts

```diff
@@ -27,21 +34,32 @@ export class OpenPGPKey {
   };
 
   public static parseMany = async (text: string): Promise<Key[]> => {
-    const result = await opgp.key.readArmored(text);
+    const resultKeys = await opgp.readKeys({ armoredKeys: text });
+    /* todo: test exception and re-wrap?
     if (result.err) {
       throw new Error('Cannot parse OpenPGP key: ' + result.err + ' for: ' + text);
     }
```

Comment:
Not checking this follows previous behavior, right? If so, we can file this for another issue, review usages, and decide how to handle these.

Reply:
Yes, this generally follow the previous behavior, but less descriptive, e.g.
`Misformed armored text` instead of `Cannot parse OpenPGP key: Misformed armored text for: ...`
I added unit tests for this.

Reply:
And also a concern about premature processing stop (same as with `KeyUtil.readBinary`)

Reply:
mentioned here https://github.com/FlowCrypt/flowcrypt-browser/pull/4725#discussion_r1090981268